lib/tasks/install-blueprint.js

```diff
@@ -80,6 +113,30 @@ class InstallBlueprintTask extends Task {
     return execa('npm', ['install'], { cwd });
   }
 
+  _npmInstallModule(module, cwd) {
+    logger.info(`Running "npm install ${module}" in "${cwd}" ...`);
+    return execa('npm', ['install', module], { cwd })
```

Comment:
Should use `yarn` if present (or based on the same rules as https://github.com/ember-cli/ember-cli/pull/6748).